sklearn/linear_model/__init__.py

```diff
@@ -30,8 +30,10 @@
 from .passive_aggressive import PassiveAggressiveClassifier
 from .passive_aggressive import PassiveAggressiveRegressor
 from .perceptron import Perceptron
+
```

Comment:
Probably when I added code for catching the exception at the time of import, after removing it, this empty line came up